console/src/components/Services/Actions/Common/components/TypeDefinitionEditor.js

```diff
@@ -52,6 +52,10 @@ const ActionDefinitionEditor = ({
 
   const errorMessage =
     error && (error.message || 'This is not valid GraphQL SDL');
+  const errorMessageLine =
+    error &&
+    error.locations &&
```

Comment:
According to the type definition of `GraphQLError` `locations` can be an empty array, so let's check for `error.locations.length` here as well.

Reply:
Oooh thanks for that!